src/web-ui/src/analytics/AnalyticsHandler.js

```diff
@@ -225,15 +225,27 @@ export const AnalyticsHandler = {
         }
     },
 
-    recordAbanonedCartEvent(user,cart) {
-        if (user && cart) {
+    recordAbanonedCartEvent(user,cart,cartProduct) {
+        if (user && cart && cartProduct) {
             AmplifyAnalytics.record({
                 name: '_session.stop',
                 attributes: {
                     HasShoppingCart: cart.items.length > 0 ? ['true'] : ['false'],
                 }
             })
-        }
+            AmplifyAnalytics.updateEndpoint({
+                userId: user.id,
+                attributes: {
+                    HasShoppingCart: cart.items.length > 0 ? ['true'] : ['false'],
+                    WebsiteCartURL : [process.env.VUE_APP_WEB_ROOT_URL + '#/cart'],
+                    WebsiteLogoImageURL : [process.env.VUE_APP_WEB_ROOT_URL + '/RDS_logo_white.svg'],
+                    WebsitePinpointImageURL : [process.env.VUE_APP_WEB_ROOT_URL + '/icon_Pinpoint_orange.svg'],
+                    ShoppingCartItemImageURL:  [process.env.VUE_APP_IMAGE_ROOT_URL + cartProduct.category + '/' + cartProduct.image],
+                    ShoppingCartItemTitle :  [cartProduct.name],
+                    ShoppingCartItemURL : [cartProduct.url],     
```

Comment:
What about including all products in cart in these attributes as multiple values and then iterating over them in the template?

Reply:
As discussed offline, I am facing some challenges in implementing the iterative approach.
Until we are able to find a solution, lets keep the template as is as it still provides a  personalzed experience.